docs/how-to/different-model.rst

```diff
@@ -0,0 +1,33 @@
+======================================
+Target a different model or controller
+======================================
+
+The current active model will be used by default, but it's possible to select a
+different one, even on a different controller. There are two ways to choose the
+model you want to operate on.
+
+Using CLI argument
+~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    $ cou plan --model <model-name>
+    $ cou plan --model <controller>:<model-name>
+
+Using environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since **COU** is using `python-libjuju`_, it's possible for some of the environment variables
+mentioned in the documented `Juju environment variables`_ to affect the behaviour of the
+program. For example, `JUJU_DATA`_ can be used to specify a different path for Juju
+configuration files.
+
+.. code::
+    
+    $ JUJU_MODEL=openstack cou plan
```

Comment:
I think that this should be removed, since the `JUJU_MODEL` is not implemented in libjuju, and we did not defined it either.